app/views/ReportsView/index.tsx

```diff
@@ -25,19 +26,20 @@ const ReportsTableComponent = (): JSX.Element => {
     onGridReady,
   } = useGrid();
 
-  const { userDetails, adminUser } = useContext(SecurityContext);
+  const { adminAccess } = useResource();
+  const isExternalMode = useExternalMode();
   const [rowData, setRowData] = useState<ReportType[]>();
 
   useEffect(() => {
-    if (!rowData) {
+    if (isExternalMode !== undefined) {
```

Comment:
should `rowData` check be removed here ...?

Reply:
No it should not, fixing that